src/views/v2/rsvp-kitchen-sink/form-not-going.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Block: RSVP form not going.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp-kitchen-sink/form-not-going.php
+ *
+ * See more documentation about our Blocks Editor templating system.
+ *
+ * @link {INSERT_ARTICLE_LINK_HERE}
+ *
+ * @var Tribe__Tickets__Ticket_Object $rsvp The rsvp ticket object.
+ *
+ * @since TBD
+ *
+ * @version TBD
+ */
+?>
```

Comment:
:no_entry_sign: There must be exactly one blank line after the file comment

<sub>hash: aa0587fa540922d601626a7f22889bb8</sub>